stylesheets/_global.scss

```diff
@@ -384,7 +384,7 @@ $avatar-size: 44px;
 }
 
 .contact-details {
-  $left-margin: 8px;
+  $left-margin: 11px;
```

Comment:
Now that React generated HTML has no spaces, we need to readjust certain styles as the spacing changes with a normalized DOM.